client/components/marketing-survey/cancel-purchase-form/step-components/test/business-at-step.jsx

```diff
@@ -1,75 +1,84 @@
 
-describe( 'BusinessATStep', () => {
-	describe( 'rendering translated content', () => {
-		let wrapper;
-		const translate = ( content ) => `Translated: ${ content }`;
-
-		beforeEach( () => {
-			wrapper = shallow( <BusinessATStep recordTracksEvent={ noop } translate={ translate } /> );
-		} );
+describe( 'rendering translated content', () => {
+	const translate = ( content ) => `Translated: ${ content }`;
 
-		test( 'should render translated heading content', () => {
-			expect( wrapper.find( 'FormSectionHeading' ).props().children ).toEqual(
-				'Translated: New! Install Custom Plugins and Themes'
-			);
-		} );
+	test( 'should render translated heading content', () => {
+		const { container } = render(
+			<BusinessATStep recordTracksEvent={ noop } translate={ translate } />
+		);
+		expect( container.firstChild ).toHaveTextContent(
+			'Translated: New! Install Custom Plugins and Themes'
+		);
+	} );
 
-		test( 'should render translated link content', () => {
-			expect( wrapper.find( 'FormFieldset > p' ).at( 0 ).props().children ).toEqual(
-				'Translated: Have a theme or plugin you need to install to build the site you want? ' +
-					'Now you can! ' +
-					'Learn more about {{pluginLink}}installing plugins{{/pluginLink}} and ' +
-					'{{themeLink}}uploading themes{{/themeLink}} today.'
-			);
-		} );
+	test( 'should render translated link content', () => {
+		render( <BusinessATStep recordTracksEvent={ noop } translate={ translate } /> );
+		expect( screen.queryByRole( 'group' ) ).toHaveTextContent(
+			'Translated: Have a theme or plugin you need to install to build the site you want? ' +
+				'Now you can! ' +
+				'Learn more about {{pluginLink}}installing plugins{{/pluginLink}} and ' +
+				'{{themeLink}}uploading themes{{/themeLink}} today.'
+		);
+	} );
 
-		test( 'should render translated confirmation content', () => {
-			expect( wrapper.find( 'FormFieldset > p' ).at( 1 ).props().children ).toEqual(
+	test( 'should render translated confirmation content', () => {
+		render( <BusinessATStep recordTracksEvent={ noop } translate={ translate } /> );
+		expect(
+			screen.getByText(
 				'Translated: Are you sure you want to cancel your subscription and lose access to these new features?'
-			);
-		} );
+			)
+		).toBeVisible();
 	} );
+} );
 
-	describe( 'rendered links', () => {
-		const translate = ( content, params ) => {
-			if ( params && params.components ) {
-				return (
-					<div>
-						{ params.components.pluginLink }
-						{ params.components.themeLink }
-					</div>
-				);
-			}
-			return null;
-		};
-		let recordTracksEvent;
-		let wrapper;
+describe( 'rendered links', () => {
+	afterEach( () => {
+		jest.clearAllMocks();
+	} );
 
-		beforeEach( () => {
-			recordTracksEvent = jest.fn();
-			wrapper = shallow(
-				<BusinessATStep translate={ translate } recordTracksEvent={ recordTracksEvent } />
+	const translate = ( content, params ) => {
+		if ( params && params.components ) {
+			return (
```

Comment:
Should we still have `content` rendered somewhere in the mock?

Reply:
This comment is still valid. If we have `components`, we'll never render the translated content in the mock, and I think we should.

Reply:
The first set of tests under 'rendering translated content' test the translated content. Would it be redundant to render the translated content in the mock in the second set of tests? Or maybe it would make sense to combine them instead?

Reply:
I think combining them would solve all the problems, yes 😉